themes/scania/c-theme.scss

```diff
@@ -203,6 +215,13 @@ h5,
 h6 {
   font-family: $headings-font-family;
   color: $headings-color;
+
+  &:first-child {
+    margin-top: 0;
```

Comment:
It removes the top margin of the header in the main page (see picture below, see *Footer* page title)
![image](https://user-images.githubusercontent.com/1199101/55003144-1972e080-4fd8-11e9-994a-0493b4dcdd57.png)

Reply:
Fix will be added in the next pull request. As a global standard, margin & padding should be provided by container